src/components/AnnotationTarget/AnnotationTarget.tsx

```diff
@@ -1,79 +0,0 @@
-import * as React from 'react';
-import classNames from 'classnames';
-import noop from 'lodash/noop';
-import scrollIntoView, { Options } from 'scroll-into-view-if-needed';
-import { KEYS } from 'box-ui-elements/es/constants';
-
-type Props = {
-    annotationId: string;
-    children: React.ReactNode;
-    className?: string;
-    isActive?: boolean;
-    onSelect?: (annotationId: string) => void;
-};
-
-const scrollOptions: Options = {
-    behavior: 'smooth',
-    block: 'center',
-    inline: 'center',
-    scrollMode: 'if-needed',
-};
-
-const AnnotationTarget = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.Element => {
-    const { annotationId, children, className, isActive, onSelect = noop, ...rest } = props;
-    const innerRef = React.useRef<HTMLAnchorElement>(null);
-
-    React.useImperativeHandle(ref, () => innerRef.current as HTMLAnchorElement, [innerRef]);
-    React.useEffect(() => {
-        if (isActive && innerRef.current) {
-            scrollIntoView(innerRef.current, scrollOptions);
-        }
-    }, [isActive]);
-
-    const cancelEvent = (event: React.SyntheticEvent): void => {
-        event.preventDefault();
-        event.stopPropagation();
-        event.nativeEvent.stopImmediatePropagation(); // Prevents document event handlers from executing
-    };
-    const handleBlur = (event: React.SyntheticEvent): void => {
-        cancelEvent(event);
-    };
-    const handleClick = (event: React.MouseEvent): void => {
-        cancelEvent(event);
-        onSelect(annotationId);
-    };
-    const handleFocus = (event: React.SyntheticEvent): void => {
-        cancelEvent(event);
-        onSelect(annotationId);
-    };
-    const handleKeyPress = (event: React.KeyboardEvent): void => {
-        if (event.key !== KEYS.enter && event.key !== KEYS.space) {
-            return;
-        }
-
-        cancelEvent(event);
-        onSelect(annotationId);
-    };
-
-    // We use an anchor to allow the target to be used inside of SVG shapes where buttons aren't supported
-    return (
-        // eslint-disable-next-line jsx-a11y/anchor-is-valid
-        <a
```

Comment:
This fun svg/anchor/rect combination will be needed if we support text highlights, but isn't necessary right now.